config/test.exs

```diff
@@ -5,7 +5,6 @@ config :elixir, :time_zone_database, Tzdata.TimeZoneDatabase
 config :tesla, adapter: Tesla.Adapter.Hackney
 
 config :caldav_client, :test_server,
-  server_url: "http://127.0.0.1:8800/cal.php",
-  auth: :basic,
+  server_url: "http://127.0.0.1:8800/dav.php",
```

Comment:
Latest Baikal image exposes CalDav endpoints at `dav.php` so I changed it here. Hopefully it's ok

Reply:
Sure! Thanks for information.